src/apple/apple-wizard.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable max-lines */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// @ts-ignore - clack is ESM and TS complains about that. It works though
+import clack from '@clack/prompts';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as xcManager from './xcode-manager';
+import * as codeTools from './code-tools';
+import * as bash from '../utils/bash';
+import { WizardOptions } from '../utils/types';
+import * as Sentry from '@sentry/node';
+import { traceStep } from '../telemetry';
+
+const xcode = require('xcode');
+/* eslint-enable @typescript-eslint/no-unused-vars */
+
+import {
+  askForProjectSelection,
+  askForSelfHosted,
+  askForWizardLogin,
+  askToInstallSentryCLI,
+  SentryProjectData,
+  printWelcome,
+  abort,
+  askForItemSelection,
+} from '../utils/clack-utils';
+
+export async function runAppleWizard(
+  options: WizardOptions,
+): Promise<void> {
+  printWelcome({
+    wizardName: 'Sentry Apple Wizard',
+    promoCode: options.promoCode,
+  });
+
+  const hasCli = bash.hasSentryCLI();
+  Sentry.setTag('has-cli', hasCli);
+  if (!hasCli) {
+    if (!(await traceStep("Ask for SentryCLI", () => askToInstallSentryCLI()))) {
+      clack.log.warn("Without sentry-cli, you won't be able to upload debug symbols to Sentry. You can install it later by following the instructions at https://docs.sentry.io/cli/");
+      Sentry.setTag('CLI-Installed', false);
+    } else {
+      await bash.installSentryCLI();
+      Sentry.setTag('CLI-Installed', true);
+    }
+  }
+
+  const projectDir = process.cwd();
+  const xcodeProjFiles = findFilesWithExtension(projectDir, ".xcodeproj");
+
+  if (!xcodeProjFiles || xcodeProjFiles.length === 0) {
+    clack.log.error('No xcode project found. Please run this command from the root of your project.');
+    await abort();
+    return;
+  }
+
+  let xcodeProjFile;
+
+  if (xcodeProjFiles.length === 1) {
+    xcodeProjFile = xcodeProjFiles[0];
+    Sentry.setTag('multiple-projects', false);
+  } else {
+    Sentry.setTag('multiple-projects', true);
+    xcodeProjFile = (await traceStep("Choose Xcode project", () => askForItemSelection(xcodeProjFiles, "Which project do you want to add Sentry?"))).value;
```

Comment:
Which project do you want to add Sentry to?
`l`:
```suggestion
    xcodeProjFile = (await traceStep("Choose Xcode project", () => askForItemSelection(xcodeProjFiles, "Which project do you want to add Sentry to?"))).value;
```